pyscf/isdf/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2014-2018 The PySCF Developers. All Rights Reserved.
```

Comment:
Please update the copyright to `2014-2024` here and for the rest of the files.

Reply:
Thanks, I will change all of them.